marketdata/stream/options.go

```diff
@@ -233,6 +237,20 @@ func WithLULDs(handler func(LULD), symbols ...string) StockOption {
 	})
 }
 
+// WithCancelErrors configures inital trade cancel errors handler
```

Comment:
Please extend the docstring with the fact that the symbols are missing from the option, because ...